charts/bpa/values.yaml

```diff
@@ -16,6 +16,9 @@ global:
   # -- The used ledger. Will be used for default values. Any of: bosch-test, idu.
   ledger: bosch-test
 
+  # -- The ux configuration. Will be used for default values. Any of: bosch, bcgov.
+  ux: bosch
```

Comment:
MAybe you can add this "default config"  under the ux path in line 161 ?

Reply:
I wanted to leave this like the schemas where there are some preconfigured values and the option to push in a different object.

Reply:
got it